SPECS/rpm/rpm.spec

```diff
@@ -19,6 +19,7 @@ Requires:       libdb
 Requires:       rpm-libs = %{version}-%{release}
 Requires:       libarchive
 Requires:       lua
+Requires:       libselinux
```

Comment:
Just checking: have we published any RPMs with the `4.14.2-11` version? Just making sure we won't be overwriting a published package.

Reply:
I don't believe we have had a successful build yet.